pkg/handler/handler_test.go

```diff
@@ -25,11 +25,15 @@ func TestRegisterOpenAPIVersionedService(t *testing.T) {
 		t.Errorf("Unexpected error in unmarshalling SwaggerJSON: %v", err)
 	}
 
-	returnedJSON, err := json.MarshalIndent(s, " ", " ")
+	returnedJSON, err := json.Marshal(s)
```

Comment:
any reason this didn't switch to jsoniter as well?

Reply:
no, just forgotten. Fixed.